docs/tables/googleworkspace_gmail_message.md

```diff
@@ -6,6 +6,8 @@ The `googleworkspace_gmail_message` table can be used to query user's messages f
 
 To list all of **your** messages use the `googleworkspace_gmail_my_message` table instead.
 
+You will almost always want to include `query=` along with a string to search for, and/or [search operators](https://support.google.com/mail/answer/7190).
```

Comment:
```suggestion
You will almost always want to include `where query=` along with a string to search for, and/or [search operators](https://support.google.com/mail/answer/7190).
```

Small addition, but thoughts on making this line consistent with the other line above a bit?

Reply:
Yup, better, thx.